tests/logout.test.js

```diff
@@ -0,0 +1,33 @@
+
+import loginPage from '../pages/login.page';
+import logoutPage from '../pages/logout.page';
+
+describe('OpenBank', () => {
+
+    it('Should show logout success message when clicking on logout menu', async () => {
+
+      await loginPage.open('login');
+      let username = 'jsmith@demo.io';
+      let password = 'Demo123!';
+      await loginPage.logIn(username, password);
+
+      await logoutPage.clickAvatar();
+      
+      expect(await browser.getUrl()).to.contain('login?logout');
```

Comment:
Consider using Webdriver.io's **matchers** whenever possible for implementing asserts that verify elements or browsers properties, as they are more idiomatic and typically shorter / more readable expressions. For this case, something like:

```javascript
expect(await browser).toHaveUrlContaining('login?logout');
```
Reference: https://webdriver.io/docs/api/expect-webdriverio/#tohaveurlcontaining

Reply:
Error:
Invalid Chai property: toHaveUrlContaining

Reply:
![image](https://user-images.githubusercontent.com/96138690/147568204-c6420333-9d77-43d8-935d-de6f0a004ac9.png)

Reply:
![image](https://user-images.githubusercontent.com/96138690/147568304-0336c67d-b1f1-4edd-b801-b295fbd5625c.png)

Reply:
What should I use: Chai or WebdriverIO??

Reply:
According to the doc you've shared, expect-webdriverio is a super-set of chai's expect, therefore by using expect-webdriverio istead of chai's you'd achieve the best of both worlds. Go with globally overriding **expect** with expect-webdriverio implementation

Reply:
Done